Extensions/News/Main.js

```diff
@@ -0,0 +1,62 @@
+const Net = require('./NetHandler');
+const fs = require('fs');
+var path = require('path');
+const { F_OK } = require('constants');
+const temp_dir = "data\\";
```

Comment:
what does the `\\` do?

Reply:
It's telling us that it is a directory, and that we are going to be working inside this directory.

If I only had the directory without the backslashes

`const temp_dir = 'data';
const temp_file = temp_dir + 'news.txt';`

The file path would result into this

`console.log(temp_file); // datanews.txt`

And that is obviously wrong, therefore I like to specify this way what is directory.